test/tools/Modules/HttpsListener/HttpsListener.psd1

```diff
@@ -0,0 +1,10 @@
+@{
+    ModuleVersion = '1.0.0'
+    GUID = '90572e25-3f15-49b0-8f25-fb717d3ef46a'
+    Author = 'Mark Kraus'
+    CompanyName = ''
+    Copyright = ''
+    Description = 'Creates a new HTTPS Listener for testing purposes'
+    RootModule = 'HttpsListener.psm1'
```

Comment:
Web listener works for me. Its more general and covers both HTTP and HTTPS. I will rename it.